tests/testflows/rbac/requirements/requirements.md

```diff
@@ -3162,7 +3202,7 @@ version: 1.0
 
 [ClickHouse] SHALL support granting or revoking **alter column** privilege
 on a specified cluster to one or more **users** or **roles**.
-Any `ALTER TABLE ... ADD|DROP|CLEAR|COMMENT|MODIFY COLUMN`
+Any `ALTER TABLE ... ADD|CREATE|CLEAR|COMMENT|MODIFY COLUMN`
```

Comment:
We don't have `ALTER TABLE ... CREATE COLUMN` command, but we have `ALTER TABLE ... DROP COLUMN` command.

Reply:
@MyroTk